vnext/src/RNTester/js/AlertExample.windows.tsx

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ * @format
+ */
+
+import React = require('react');
```

Comment:
>require [](start = 15, length = 7)

Do we need to create a windows fork for the AlertExample? What is the reason we are missing const example definition in the first place? Is it an update to RNTester that we did not pick up in Microsoft Fork? If so, we can probably just update the AlertExample script in RNTesterCopy directory.

Reply:
I would be interested in whether this issue is upstream RNTester. If it is, and is still broken, we should make the fix there. If it was broken there then fixed, I'd rather we just wait for the fix than do more local patching. 

If it is, I'd almost prefer we make the change there and just wait for the fix

Reply:
I would be interested in whether this issue is upstream RNTester. If it is, and is still broken, we should make the fix there. If it was broken there then fixed, I'd rather we just wait for the fix than do more local patching. 

If it is, I'd almost prefer we make the change there and just wait for the fix

Reply:
+1 let's try to avoid forking.  FWIW this test did work when it was first written.

---
In reply to: [372745902](https://github.com/microsoft/react-native-windows/pull/3992#discussion_r372745902) [](ancestors = 372745902)

Reply:
I've removed the Alert fork for now to unblock the Text page. I will create a new PR if there's no repro on Android.